README.md

```diff
@@ -2,3 +2,7 @@
 
 ## Why?
 I'm a consultant, software engineer, cloud architect, devops pro and I can't remember everything. This is the swap file for my brain. I need to store, access, and quickly retrieve what I have learned!
+
+## How to run in DEV mode
```

Comment:
@laurelherting need to use two terminals now. One to run yarn start and the other for dev-server that serves up that `db.json` file